### DISCUSSION TITLE
Xaml Hot Reload for ApplicationTemplate on mobile net7 is not working.

### DISCUSSION THREAD

**Discussion by LProulx86** [unoplatform/uno#14216] [Q&A]
Hot reload does not activate for Android and iOS in [UnoApplicationTemplate](https://github.com/nventive/UnoApplicationTemplate).

When looking for `RemoteControlGenerator\RemoteControl.g.cs` the folder and file are simply not generated.

I look on [working-with-xaml-hot-reload](https://platform.uno/docs/articles/features/working-with-xaml-hot-reload.html), but didn't find any info about my problem.

Live Visual Tree : "XAML Hot Reload is ether disable, not supported by the current application or the application execution is paused."

I have VS 17.7.6 and test on latest VS preview too (17.8.0 Preview 6.0).

What it is required to activate XAML Hot Reload on Android and iOS net7 ?

Thank you.

**Comment by jeromelaban** [comment]
We've published an updated for Hot Reload on iOS and Android with Uno 5.0. There are still some issues for iOS/Android/Catalyst where the runtime crashes (https://github.com/dotnet/runtime/issues/93860), but XAML Hot Reload only should be working fine.

You'll find additional troubleshooting that we published yesterday [here](https://platform.uno/docs/articles/features/working-with-xaml-hot-reload.html).
